lldb/packages/Python/lldbsuite/test/gdbclientutils.py

```diff
@@ -374,15 +378,17 @@ class UnexpectedPacketException(Exception):
         pass
 
 
-class ServerChannel:
+class ServerChannel(ABC):
```

Comment:
This is not "adding type hints".

"Change the title to "Add type hints and use abstract base class in"

Reply:
Done and just to confirm abc was in python 3.4 so it's not part of the 3.8/3.9 issue

Reply:
Thanks, good to know.